operator/internal/manifests/openshift/opa_openshift.go

```diff
@@ -38,14 +38,17 @@ func newOPAOpenShiftContainer(mode lokiv1.ModeType, secretVolumeName, tlsDir, mi
 	uriScheme = corev1.URISchemeHTTP
 	args = []string{
 		"--log.level=warn",
-		"--opa.skip-tenants=audit,infrastructure",
-		"--opa.admin-groups=system:cluster-admins,cluster-admin,dedicated-admin",
 		fmt.Sprintf("--web.listen=:%d", GatewayOPAHTTPPort),
 		fmt.Sprintf("--web.internal.listen=:%d", GatewayOPAInternalPort),
 		fmt.Sprintf("--web.healthchecks.url=http://localhost:%d", GatewayOPAHTTPPort),
+		"--opa.skip-tenants=audit,infrastructure",
```

Comment:
Do we want to keep the constant arguments at the top?

Reply:
I'm more interested in keeping in keeping arguments that are related together (opa, web.. etc)